pkger/models.go

```diff
@@ -1415,6 +1455,163 @@ func (n mapperNotificationEndpoints) Len() int {
 	return len(n)
 }
 
+const (
+	fieldNotificationRuleCurrentLevel    = "currentLevel"
+	fieldNotificationRuleEndpointName    = "endpointName"
+	fieldNotificationRuleMessageTemplate = "messageTemplate"
+	fieldNotificationRulePreviousLevel   = "previousLevel"
+	fieldNotificationRuleStatusRules     = "statusRules"
+	fieldNotificationRuleTagRules        = "tagRules"
+)
+
+type notificationRule struct {
+	name         string
+	description  string
+	endpointName string
+	every        time.Duration
+	msgTemplate  string
+	offset       time.Duration
+	status       string
+	statusRules  []struct{ curLvl, prevLvl string }
+	tagRules     []struct{ k, v, op string }
+
+	labels sortedLabels
+}
+
+func (r *notificationRule) Name() string {
+	return r.name
+}
+
+func (r *notificationRule) ResourceType() influxdb.ResourceType {
+	return KindNotificationRule.ResourceType()
+}
+
+func (r *notificationRule) Status() influxdb.Status {
+	if r.status == "" {
```

Comment:
When would status be unset for notification rules?

Reply:
good question. After speaking with @russorat, we decided that in pkger, you don't have to provide a status, it'll just default to `active` if not provided.

Reply:
got it, that makes sense